src/components/Editor/ItemEditor.vue

```diff
@@ -575,7 +618,7 @@ export default {
     },
     charLimitBoxClass() {
       // class for the input area to enter max char limit
-      return "text-center disabled:opacity-50";
+      return " text-sm text-center disabled:opacity-50";
```

Comment:
have you checked for responsiveness? is text-sm the right value across all screen sizes?

Reply:
yes checked